telegram/_chatfullinfo.py

```diff
@@ -0,0 +1,166 @@
+        for backwards compatibility. In the future, this class will *NOT* inherit from
+        :class:`telegram.Chat`.
+
+    .. seealso::
+        All arguments and attributes can be found in :class:`telegram.Chat`.
+
+    Args:
+        max_reaction_count (:obj:`int`): The maximum number of reactions that can be set on a
+            message in the chat.
+
+            .. versionadded:: NEXT.VERSION
+
+    Attributes:
+        max_reaction_count (:obj:`int`): The maximum number of reactions that can be set on a
+            message in the chat.
+
+            .. versionadded:: NEXT.VERSION
+    """
+
+    __slots__ = ("max_reaction_count",)
+
+    def __init__(
+        self,
+        id: int,
+        type: str,
+        accent_color_id: int,  # API 7.3 made this argument required
+        max_reaction_count: int,  # NEW arg in api 7.3 and is required
+        title: Optional[str] = None,
+        username: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None,
+        photo: Optional[ChatPhoto] = None,
+        description: Optional[str] = None,
+        invite_link: Optional[str] = None,
+        pinned_message: Optional["Message"] = None,
+        permissions: Optional[ChatPermissions] = None,
+        sticker_set_name: Optional[str] = None,
+        can_set_sticker_set: Optional[bool] = None,
+        slow_mode_delay: Optional[int] = None,
+        bio: Optional[str] = None,
+        linked_chat_id: Optional[int] = None,
+        location: Optional[ChatLocation] = None,
+        message_auto_delete_time: Optional[int] = None,
+        has_private_forwards: Optional[bool] = None,
+        has_protected_content: Optional[bool] = None,
+        join_to_send_messages: Optional[bool] = None,
+        join_by_request: Optional[bool] = None,
+        has_restricted_voice_and_video_messages: Optional[bool] = None,
+        is_forum: Optional[bool] = None,
+        active_usernames: Optional[Sequence[str]] = None,
+        emoji_status_custom_emoji_id: Optional[str] = None,
+        emoji_status_expiration_date: Optional[datetime] = None,
+        has_aggressive_anti_spam_enabled: Optional[bool] = None,
+        has_hidden_members: Optional[bool] = None,
+        available_reactions: Optional[Sequence[ReactionType]] = None,
+        background_custom_emoji_id: Optional[str] = None,
+        profile_accent_color_id: Optional[int] = None,
+        profile_background_custom_emoji_id: Optional[str] = None,
+        has_visible_history: Optional[bool] = None,
+        unrestrict_boost_count: Optional[int] = None,
+        custom_emoji_sticker_set_name: Optional[str] = None,
+        birthdate: Optional[Birthdate] = None,
+        personal_chat: Optional["Chat"] = None,
+        business_intro: Optional["BusinessIntro"] = None,
+        business_location: Optional["BusinessLocation"] = None,
+        business_opening_hours: Optional["BusinessOpeningHours"] = None,
```

Comment:
just wondering: are the arguments here sorted like in the TG docs? since we're creating a new class here, that would be a good chance.